web/client/epics/contextcreator.js

```diff
@@ -75,23 +75,23 @@ export const saveContextResource = (action$, store) => action$
     .exhaustMap(({destLocation}) => {
         const state = store.getState();
         const resource = resourceSelector(state);
-        const context = newContextSelector(state);
         const newResource = generateContextResource(state);
+        const destLocations = {manager: '/context-manager', context: `/context/${resource.name}`};
         return (resource && resource.id ? updateResource : createResource)(newResource)
             .switchMap(rid => Rx.Observable.merge(
                 // LOCATION_CHANGE triggers notifications clear, need to work around that
                 // can't wait for CLEAR_NOTIFICATIONS, because either in firefox notification action doesn't trigger
                 // or in chrome it triggers too early
                 // (on chrome there is another LOCATION_CHANGE after the first one for unknown reason, that cancels out the first)
-                (destLocation === '/context-manager' ? action$.ofType(CONTEXTS_LIST_LOADED).take(1).switchMap(() => Rx.Observable.of(
+                (destLocation === destLocations.manager ? action$.ofType(CONTEXTS_LIST_LOADED).take(1).switchMap(() => Rx.Observable.of(
                     show({
                         title: "saveDialog.saveSuccessTitle",
                         message: "saveDialog.saveSuccessMessage"
                     }))) : Rx.Observable.empty()),
                 Rx.Observable.of(
                     contextSaved(rid),
-                    push(destLocation || `/context/${context.name}`),
-                    loadExtensions(),
+                    push(destLocation || destLocations.context),
```

Comment:
If we remove the action we have this problem now:

https://github.com/user-attachments/assets/f867624e-8f64-4f4b-bc2b-3765eeb17dbf

Unfortunately my guess on the `loadExtensions` was wrong and this function is needed because it adds the new uploaded extensions in the page and it triggers a full application reload, so it's not possible to add it to an internal workflow at the moment (eg. while uploading an extension).
Not sure if we can check somewhat that a new extension has been uploaded during the current context creator save session and only in that case trigger the `loadExtensions` function?